sdk/metrics-testing/build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id("otel.java-conventions")
+    id("otel.publish-conventions")
+}
+
+description = "OpenTelemetry SDK Testing utilities"
+otelJava.moduleName.set("io.opentelemetry.sdk.metrics-testing")
```

Comment:
We've generally suffixed with `metrics` in all our packages (even had a gradle hack that matched the suffix before the more declarative `gradle.properties` :P). Since alpha and eventually will be merged into testing doesn't matter too much though

Reply:
If possible, I'd like to avoid chaning the module name (and directory) because I have a slew of PRs lined up behind this one for the new metrics implementation and the least amount of major rebasing I need to do, the better.

If this were a fundamental naming issue (or something that wouldn't be resolved when metrics goes stable) I'd be happy to change it, but as things are my commits are starting to get unwieldy if I leave this two long (hence trying to piece out components like this one for review).

Reply:
we can always go back and change module names at the end of the PR chain, if anyone feels strongly about it.